### DISCUSSION TITLE
Formula 1 API

### DISCUSSION THREAD

**Discussion by toquete** [BrightDV/BoxBox#93] [Q&A]
I am writing my own Formula 1 stats app. I already use the Ergast API, but I want to display more information than it provides me. I was searching for another API and came to this repo.

How do you get access to the http://api.formula1.com?

**Comment by BrightDV** [comment]
I don't think that this API would be useful to you: I use it only to retrieve the news (but I don't know all the existing endpoints).
If you want to use it, check the news.dart file (lib > api).
Otherwise, there are amazing websites for stats like https://statsf1.com but you may need to scrape it.
